dev-support/docker-compose/scripts/start-httpfs.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+sleep 5
```

Comment:
why are you sleeping 5 seconds? Does start script fails if namenode hasn't started yet?

Reply:
No, it is just for the docker-compose log output to be more organized. I plan to circle back to this at a later time